### ISSUE TITLE
Should Ostinato VM include the GUI ?

### ISSUE THREAD

**Issue by ehlers** [GNS3/gns3-registry#62]
I'm just preparing the next-gen ostinato VM, see https://github.com/ehlers/gns3-registry/commit/25ad2334684e0f0fea70385dbb9840814f6605db. It's for the upcoming 0.8 release of ostinato, which will probably take some time.

Now I'm thinking of additionally including the GUI in the VM. That will make it much easier to use it for the occasional user. But of course it will increase the size of the VM. The experienced user can still use an ostinato GUI on his local OS.

@noplay, what do you think?

**Comment by boenrobot** [comment]
Why not just have both versions in the registry? "0.8" and "0.8 (with GUI)".

**Comment by noplay** [comment]
I think adding the GUI will make it much much simpler to use.

Le mer. 30 déc. 2015 16:44, Vasil Rangelov <notifications@github.com> a
écrit :

**Comment by ehlers** [comment]
I decided to include the GUI in the ostinato VM, it makes to really much simpler to use. As version 0.8 seems to take quite some time, I created an updated VM for the current version 0.7.1.

I was also thinking about a drone-only VM, but it's just about 20 MB smaller as the full VM and doesn't have any advantage. The remote control of the drone is also possible with the full VM. As I don't want to support two VMs, I won't offer a pre-build drone-only VM. Nevertheless I created a packer script also for the drone-only VM, so everyone can build it.

Tomorrow I will update the documentation, then I will create a PR.
